parsers/s01-parse/Azlaroc/sftpgo-logs.yaml

```diff
@@ -0,0 +1,29 @@
+onsuccess: next_stage
+pattern_syntax:
+  SFTPGO_TIME: '%{YEAR}-%{MONTHNUM}-%{MONTHDAY}T%{HOUR}:%{MINUTE}:%{SECOND}\.%{NUMBER}'
+  SFTPGO_FAILED: '\{"level":"%{WORD:log_level}","time":"%{SFTPGO_TIME:evt_time}","sender":"connection_failed","client_ip":"%{IPV4:client_ip}","username":"%{DATA:username}","login_type":"%{DATA:login_type}","protocol":"%{WORD:protocol}","error":"%{GREEDYDATA:error}"\}'
+filter: evt.Line.Labels.type == 'sftpgo'
+nodes:
+  - grok:
+      name: SFTPGO_FAILED
+      apply_on: Line.Raw
```

Comment:
`message`

to allow s00 processor to set the `evt.Parsed.message` attribute as `raw` may have `syslog` or other s00 prefixes that need to be stripped.